runtime/constants.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  AppScheduler: {
+    status: {
+      notRunning: 'not running',
+      creating: 'creating',
+      running: 'running',
+      destructing: 'destructing',
+      exited: 'exited'
+    },
+    modes: {
+      default: 0, // 0b00
+      test: 1, // 0b01
+      debug: 2 // 0b10
```

Comment:
What's the meaning of these comments?

Reply:
BTW, this file's name is constant, so the following might be alternatives:

1. rename the file, the name implies this file could be programable
2. use JSON instead of JS, once we have the programmatic requirements

Reply:
> Currently there is no support for syntax 0b01 in ShadowNode.

Opened an issue https://github.com/Rokid/ShadowNode/issues/446, and could we use hex instead for now?

> JSON doesn't support binary number representations too.

Reasonable and looks good to me.

Reply:
I don't get the approaches to represent binary flags using hex.